### PR TITLE
Bugfix: PHP7 Fixes

### PR DESCRIPTION
**Pull request by brianmuse**
Fixes issues with PHP-AR and PHP7

See "Changes to the handling of indirect variables, properties, and methods" on: http://php.net/manual/en/migration70.incompatible.php

See also: 
- https://github.com/jpfuentes2/php-activerecord/commit/8c907fe116cd34ef9da6403570287ff6d7e91f08
- https://github.com/jpfuentes2/php-activerecord/pull/553

### CODE REVIEW

File: lib/Model.php [hunk 1]
```diff
@@ -415,7 +415,7 @@ public function __set($name, $value)
 		foreach (static::$delegate as &$item)
 		{
 			if (($delegated_name = $this->is_delegated($name,$item)))
-				return $this->$item['to']->$delegated_name = $value;
+				return $this->{$item['to']}->$delegated_name = $value;
```
Comment (Rican7) [inline lib/Model.php]: [I mentioned this in the upstream PR](https://github.com/jpfuentes2/php-activerecord/pull/553#discussion_r88141490), but we should always be using the curly-brace wrappers whenever we nest references like this. I'd suggest changing this to:

``` php
return $this->{$item['to']}->{$delegated_name} = $value;
```

**Review by Rican7 — COMMENTED** [review]
One tiny little change suggestion here. Otherwise this is great! 😃

File: lib/Model.php (hunk 1 above)
Comment (brianmuse) [reply lib/Model.php]: Yeah may as well. Did a search proj-wide and there's a tonnnnn of these. Butmay as well driveby fix here 👍

**Review by jstruzik — APPROVED** [review]
I'm good on this PR regardless of what happens in https://github.com/robinpowered/php-activerecord/pull/12/files#r88142204

File: lib/Model.php (hunk 1 above)
Comment (Rican7) [reply lib/Model.php]: Lol, of course there is. 😞

**Review by Rican7 — APPROVED** [review]
Looks great Brian! Thanks for fixing the compatibility bug. I'm psyched for potential PHP 7 support!

**Review by jstruzik — APPROVED** [review]

**Pull request merged**